src/System.Diagnostics.Process/src/System/Diagnostics/Process.FreeBSD.cs

```diff
@@ -64,8 +65,29 @@ public TimeSpan PrivilegedProcessorTime
         }
 
         /// <summary>Gets parent process ID</summary>
-        private int ParentProcessId =>
-            throw new PlatformNotSupportedException();
+        private unsafe int ParentProcessId
+        {
+            get
+            {
+                EnsureState(State.HaveNonExitedId);
+
+                Interop.Process.kinfo_proc* processInfo = null;
+                try
+                {
+                    processInfo = Interop.Process.GetProcInfo(_processId, false, out var count);
```

Comment:
This doesn't need to be in the try. You can just declare/initialize it above.

Reply:
Nit: var => whatever the type is

Reply:
There is a validation which throws `ArgumentOutOfRangeException` in case the size != struct size after the syscall. In the other two usages of this method in [FreeBSD/Interop.Process.cs](https://github.com/am11/corefx/blob/871cf282237f8f832408e69aaa4bd9b8602d0f0b/src/Common/src/Interop/FreeBSD/Interop.Process.cs), it was in the try block. Should we keep it consistent?

Updated `var` -> `int`